runners/flink/src/main/java/org/apache/beam/runners/flink/FlinkBatchTranslationContext.java

```diff
@@ -57,9 +57,12 @@
 
   private AppliedPTransform<?, ?, ?> currentTransform;
 
+  private final CountingPipelineVisitor countingPipelineVisitor = new CountingPipelineVisitor();
```

Comment:
This is needed in upcoming PR to resolve number of consumers of GBK transforms, so we can decide whether to materialize the result before sending downstream.